var/spack/repos/builtin/packages/curl/package.py

```diff
@@ -286,9 +286,6 @@ class Curl(NMakePackage, AutotoolsPackage):
     depends_on("libssh", when="+libssh")
     depends_on("krb5", when="+gssapi")
 
-    # curl queries pkgconfig for openssl compilation flags
-    depends_on("pkgconfig", type="build")
```

Comment:
This was unintentional?

Reply:
No, that line is a duplicate of the `depends_on` that is conditional on platform as `pkgconfig` is not available on native Windows and we do not support Mingw/cygwin/Msys as "Windows" platforms.

Reply:
See https://github.com/spack/spack/pull/38756/files/ff0fccbe3bd8f6371c9f773e7d9ba16570b0971e#diff-d252ddd993bccb99a66c45999d873a6ad08ce080df753b718838e34fc071ec64L272

Reply:
Oof, thanks